core/documentation/introduction/Welcome.mdx

```diff
@@ -1,8 +1,21 @@
 import { Meta } from "@storybook/blocks";
+import json from "../../../package.json";
 
-<Meta title="Documentation/Introduction/Introduction" />
+<Meta title="Documentation/Introduction/Welcome" />
 
-# Introduction
+<div class="full-width-bg">
+# Core Style Library
+
+</div>
+
+<div class="version-badges">
+  <span class="fudis-badge fudis-badge__primary">
```

Comment:
It seems that these `fudis-badge__primary` classes are redundant.

Reply:
I left these to resemble the actual badge styles (same in Ngx project),  but yeah since they are heavily overriden maybe it doesn't have to stick to the same class structure.

Reply:
I left these styles cause they actually work, but storybook documentation badge adds a bunch of styles that override these. So the styles pretty much have be written custom again. I felt it would be better to have the same class names at least. But since you feel these are unnecessary I will remove them.